scripts/utils/render.js

```diff
@@ -86,15 +91,30 @@ const packageCardBuilder = (array) => {
   </div>`;
   });
 
-  domString += "</div>"
-
-  renderToDom("#mainContentDiv", domString);
+  domString += '</div>';
 
+  renderToDom('#mainContentDiv', domString);
 };
 
 /* Form Builders */
-const overviewFormBuilder = () => {
-  return;
+const overviewFormBuilder = (repoArray) => {
+  const domString = `
+  <form id="pinRepoForm">
+    <div class="mb-3">
+      <label for="pinRepoSelect" class="form-label"><h2>Pin a Repo</h2></label>
+      <select required id="pinRepoSelect" class="form-control form-select form-select-lg" aria-label="Pin Repo Select">
+        <option value="" selected>Select a Repo</option>
+        ${repoArray
+          .map((r, index) => ({ ...r, index }))
```

Comment:
Here I add the index to the element so that I can reference it later on the pin event